packages/api/src/beacon/routes/lightclient.ts

```diff
@@ -91,7 +92,18 @@ export type Endpoints = {
   >;
 };
 
-export function getDefinitions(_config: ChainForkConfig): RouteDefinitions<Endpoints> {
+export function getDefinitions(config: ChainForkConfig): RouteDefinitions<Endpoints> {
+  // Cache config so fork digests don't need to be recomputed
+  let beaconConfig: BeaconConfig | undefined;
+
+  const cachedBeaconConfig = (): BeaconConfig => {
+    if (beaconConfig === undefined) {
+      const genesisValidatorsRoot = genesisData[config.CONFIG_NAME as NetworkName]?.genesisValidatorsRoot;
```

Comment:
Caveat with this is that custom networks won't be supported across different clients, however, as long as Lodestar is both the client and server this still works as both use zero hash as genesis validators root.

There is also always the option to use JSON encoding, so I don't think this is a big issue.